### DISCUSSION TITLE
Updating Photoprism

### DISCUSSION THREAD

**Discussion by xhabih** [photoprism/photoprism#2309] [Q&A]
I'm trying to update my photoprism on my raspberry.
I do:
> sudo docker-compose pull

I get:
> Pulling photoprism ... done
> Pulling mariadb    ... done

I restart but I'm still on 210523-b1856b9d-Linux-aarch64
I'm missing something?

**Comment by graciousgrey** [comment]
In your docker-compose.yml, do you use
 `image: photoprism/photoprism:latest` ?

--> https://docs.photoprism.app/getting-started/raspberry-pi/#modern-arm64-based-devices

**Reply by xhabih** [reply]
I have:
`image: photoprism/photoprism-arm64:latest`

**Reply by graciousgrey** [reply]
This is outdated we have multi-arch images now. 
Change it to photoprism/photoprism:latest and it should work :)

https://hub.docker.com/r/photoprism/photoprism-arm64

**Reply by xhabih** [reply]
Done, but now I get

```
sudo docker-compose pull
Pulling photoprism ... pulling from photoprism/photoprism
Pulling mariadb    ... done

ERROR: for photoprism  no matching manifest for linux/arm/v7 in the manifest list entries
ERROR: no matching manifest for linux/arm/v7 in the manifest list entries

```

**Reply by rorymurtagh** [reply]
The fix for that for a Pi4 at least add:
    platform: "linux/arm64"

So it looks like this:
services:
  ## App Server (required)
  photoprism:
    ## Use photoprism/photoprism:preview for testing preview builds:
    image: photoprism/photoprism:latest
    platform: "linux/arm64"
    depends_on:
      - mariadb

Hopefully you already have:
arm_64bit=1

In your boot/config.txt

**Reply by xhabih** [reply]
Done and working.
Thanks!
